src/main/java/com/creativemd/littletiles/LittleTiles.java

```diff
@@ -60,20 +62,29 @@ public class LittleTiles {
     public static final String modid = "littletiles";
     public static final String version = LTTags.VERSION;
 
-    public static int maxNewTiles = 512;
-
-    public static BlockTile blockTile = new BlockTile(Material.rock);
-    public static Block coloredBlock = new BlockLTColored().setBlockName("LTBlocks");
-
-    public static Item hammer = new ItemHammer().setUnlocalizedName("LTHammer");
-    public static Item recipe = new ItemRecipe().setUnlocalizedName("LTRecipe");
-    public static Item multiTiles = new ItemMultiTiles().setUnlocalizedName("LTMultiTiles");
-    public static Item saw = new ItemLittleSaw().setUnlocalizedName("LTSaw");
-    public static Item container = new ItemTileContainer().setUnlocalizedName("LTContainer");
-    public static Item wrench = new ItemLittleWrench().setUnlocalizedName("LTWrench");
-    public static Item chisel = new ItemLittleChisel().setUnlocalizedName("LTChisel");
-    public static Item colorTube = new ItemColorTube().setUnlocalizedName("LTColorTube");
-    public static Item rubberMallet = new ItemRubberMallet().setUnlocalizedName("LTRubberMallet");
+    public static int maxNewTiles = 51200;
+
+    public static CreativeTabs creativeTabLittleTiles = new LittleTilesCreativeTab("LTTab");
```

Comment:
Would be better to use `littletiles` instead of `LTTab` here to reduce confusion

Reply:
Sure, done